genomic_data_service/region_indexer.py

```diff
@@ -19,6 +20,10 @@
     'chr18', 'chr19', 'chr20', 'chr21', 'chr22', 'chrX', 'chrY'
 ]
 
+ENCODE_DOMAIN = 'https://test.encodeproject.org'
```

Comment:
this needs to be https://www.encodeproject.org;  The test machine is https://test.encodedcc.org anyway (but it redirects)

Reply:
I had it pointing to test so it doesn't put too much stress on prod. I will switch back